Mathlib/RepresentationTheory/Action.lean

```diff
@@ -509,6 +509,16 @@ theorem tensorHom {W X Y Z : Action V G} (f : W ⟶ X) (g : Y ⟶ Z) : (f ⊗ g)
 set_option linter.uppercaseLean3 false in
 #align Action.tensor_hom Action.tensorHom
 
+@[simp]
+theorem whiskerLeft_v (X : Action V G) {Y Z : Action V G} (f : Y ⟶ Z) :
```

Comment:
I'm not sure the `v` is appropriate here, I'd expect `hom`

Reply:
Thanks! I also renamed `tensorHom` to `tensor_hom` (probably a remnant from porting).